lib/cmd/common/local.js

```diff
@@ -66,127 +67,20 @@ var doLogHighlight = false;
 fs.exists = fs.exists || path.exists;
 fs.existsSync = fs.existsSync || path.existsSync;
 
-var defaultArgs = {
-  fh3: {
-    required: true,
-    val: "false",
-    type: "boolean"
-  },
-  clientDir: {
-    required: true,
-    val: "./client",
-    type: "string"
-  },
-  clientHost: {
-    required: true,
-    val: "http://127.0.0.1",
-    type: "string"
-  },
-  clientPort: {
-    required: true,
-    val: 8000,
-    type: "number"
-  },
-  cloudDir: {
-    required: true,
-    val: "./cloud",
-    type: "string"
-  },
-  cloudHost: {
-    required: true,
-    val: "http://127.0.0.1",
-    type: "string"
-  },
-  cloudPort: {
-    required: true,
-    val: 8001,
-    type: "number"
-  },
-  startCloud: {
-    required: true,
-    val: true,
-    type: "boolean"
-  },
-  debug: {
-    required: false,
-    val: false,
-    type: "boolean"
-  },
-  debugBrk: {
-    required: false,
-    val: false,
-    type: "boolean"
-  },
-  redisHost: {
-    required: true,
-    val: "127.0.0.1",
-    type: "string"
-  },
-  redisPort: {
-    required: true,
-    val: 6379,
-    type: "number"
-  },
-  redisPassword: {
-    required: false,
-    val: "",
-    type: "string"
-  },
-  usePackages: {
-    required: false,
-    val: true,
-    type: "boolean"
-  },
-  packages: {
-    required: false,
-    val: "",
-    type: "string"
-  },
-  localDB: {
-    required: true,
-    val: "true",
-    type: "boolean"
-  },
-  decoupled: {
-    required: false,
-    val: false,
-    type: "boolean"
-  },
-  logprefix: {
-    required: true,
-    val: "false",
-    type: "boolean"
-  },
-  loghighlight: {
-    required: true,
-    val: "true",
-    type: "boolean"
-  },
-  openClient: {
-    required: true,
-    val: "false",
-    type: "boolean"
-  }
-};
-
-fs.exists = fs.exists || path.exists;
-fs.existsSync = fs.existsSync || path.existsSync;
+var defaultArgs = require('./data/localDefaultArgs');
 
 function validateArgs(args) {
-  for (var i in defaultArgs) {
-    var defaultArg = defaultArgs[i],
-      currentArg = args[i];
-
-    if (args[i] === undefined) {
-      if (defaultArg.required) {
-        if (!defaultArg.val) {
-          throw new Error("Missing '" + i + "' parameter");
-        } else if (defaultArg.val) {
-          currentArg = args[i] = defaultArg.val;
-        }
-      }
+  _.each(defaultArgs, function (defaultArg, i) {
+    var currentArg = args[i];
+    if (typeof args[i] === 'undefined' && defaultArg.required && !defaultArg.val) {
+      throw new Error("Missing '" + i + "' parameter");
+    } else if (defaultArg.val) {
+      currentArg = args[i] = defaultArg.val;
     }
-    if (currentArg !== undefined) {
+
+
+
+    if (currentArg) {
```

Comment:
I'm not sure why those ended up like that. Eslint has [no-multiple-empty-lines](http://eslint.org/docs/rules/no-multiple-empty-lines) that we should also give a try.